1.9/api/agent-routes.md

```diff
@@ -0,0 +1,16 @@
+---
+post_title: Agent Routes
+menu_order: 2
+---
+
+Admin Router Agent runs on DC/OS agent nodes and exposes the following API routes, grouped by the component to which they proxy or relate.
```

Comment:
I changed the grouping. so this grouping phrase no longer applies. I'll delete it.